src/libs/IrcClient.js

```diff
@@ -633,6 +644,7 @@ function clientMiddleware(state, network) {
                         username: user.ident || undefined,
                         away: user.away ? 'Away' : '',
                         realname: user.real_name,
+                        account: user.account === 0 ? '' : user.account,
```

Comment:
Why would `user.account` ever be `0`?

Reply:
user account is 0 when there is no account, as whox is space separated it has to return something.
i wasnt sure if this should be in irc-fw or here

Reply:
Then that should be checked in irc-fw and either not set the account on the event, or set it as an empty string